docs/user-guide/global-configurations/deployment-charts.md

```diff
@@ -73,107 +97,145 @@ You can use the following variables in the helm template (all the placeholders a
 
-![Uploading custom chart](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/List+-+Empty-4.png)
+* Click **Upload Chart**.
 
-### Validation
+    ![Figure 4: Upload Chart Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-charts/upload-chart.jpg)
 
-The uploaded archive will be validated against:
+* Click **Select .tgz file** and upload your packaged deployment chart (in **.tgz** format).
 
-- Supported archive template should be in `*.tgz` format.
-- Content of `values.yaml` should be there in `app-values.yaml` file.
-- `release-values.yaml` file is required.
-- ConfigMap/Secret template should be same as that of our [reference chart](https://github.com/devtron-labs/devtron/tree/main/scripts/devtron-reference-helm-charts/reference-chart_4-14-0).
-- `Chart.yaml` must include the name and the version number.
-- `..image_descriptor_template.json` file should be present and the field format must match the format listed in the image builder template section.
+    ![Figure 5: Uploading .tgz File](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-charts/select-tgz-file.jpg)
 
+The system initiates the validation of your uploaded chart. You may also click **Cancel upload** if you wish to abort the process.
 
-The following are the validation results:
+![Figure 6: Cancelling Upload](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-charts/cancel-upload.jpg)
 
-| Validation status | Description | User action |
-| :--- | :--- | :--- |
-| **Success** | The files uploaded are validated. | Enter a description for the chart and select **Save** or **Cancel upload**. |
-| **Unsupported template** | The archive file do not match the [required template](#prerequisites). | **Upload another chart** or **Cancel upload**. |
-| **New version detected** | You are uploading a newer version of an existing chart | Enter a **Description** and select **Save** to continue uploading, or **Cancel upload**. |
-| **Already exists** | There already exists a chart with the same version. | <ul><li>Edit the version and re-upload the same chart using **Upload another chart**.</li><li>Upload a new chart with a new name using  **Upload another chart**.</li><li>**Cancel upload**.</li></ul> |
+### Validation Checks
 
-![Chart validated](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/List+-+Empty-2.png)
+In the uploading process, your file will be validated against the following criteria:
 
-![Unsupported template](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/List+-+Empty.png)
+- Supported archive template should be in `*.tgz` format.
+- ConfigMap/Secret template should be the same as that of our [reference chart](https://github.com/devtron-labs/devtron/tree/main/scripts/devtron-reference-helm-charts/reference-chart_4-14-0).
```

Comment:
I doubt if we do validate the cm/secret template

Reply:
Correct, we don't validate